src/metpy/plots/declarative.py

```diff
@@ -987,8 +996,9 @@ def draw(self):
             if getattr(self, 'handle', None) is None:
                 self._build()
             if getattr(self, 'colorbar', None) is not None:
-                self.parent.ax.figure.colorbar(
-                    self.handle, orientation=self.colorbar, pad=0, aspect=50)
+                self.parent.ax.figure.colorbar(self.handle, orientation=self.colorbar,
+                                               pad=0, aspect=50)\
+                    .ax.tick_params(labelsize=self.colorbar_fontsize)
```

Comment:
I really don't like Python's line continuation characters. How about:
```suggestion
                cbar = self.parent.ax.figure.colorbar(
                    self.handle, orientation=self.colorbar, pad=0, aspect=50)
                cbar.ax.tick_params(labelsize=self.colorbar_fontsize)
```